driver-core/src/test/functional/com/mongodb/OperationFunctionalSpecification.groovy

```diff
@@ -264,12 +264,18 @@ class OperationFunctionalSpecification extends Specification {
             getReadConnectionSource() >> connectionSource
             getReadPreference() >> readPreference
             getSessionContext() >> Stub(SessionContext) {
+                hasSession() >> true
                 hasActiveTransaction() >> false
                 getReadConcern() >> readConcern
             }
         }
         def writeBinding = Stub(WriteBinding) {
             getWriteConnectionSource() >> connectionSource
+            getSessionContext() >> Stub(SessionContext) {
```

Comment:
The changes to the Stubs are necessary because now there are unit tests that are calling (indirectly) SessionContext.hasSession() and require it to return true, rather than the default (false) which Spock will do without this change.

Reply:
Ack